ansible/roles/web/tasks/php.yml

```diff
@@ -31,7 +34,10 @@
 
 - name: Install packaged PHP extensions for development
   become: true
-  apt:  package=php{{php_version}}-{{ item }} state=present
+  apt:
+    package:
+        - php{{php_version}}-{{ item }}
+    state: present
```

Comment:
That nicely works around the deprecation but doesn't solve the issue of combining all packages into a single `apt` call in the future (That's what the warning is all about). Anyways, let's take it for now, all other options are extremely ugly.